frontend/pages/index.js

```diff
@@ -132,27 +299,59 @@ const ImpactSection = () => {
 
 const PartnersSection = () => {
```

Comment:
This should be it's own file, as should compensators. Both should use the same generic component. I like a convention of having a folder of "business logic" or "domain" specific components which are mostly containers for generic components but contain all business logic (or even better, delegate the logic to hooks ) and then a folder of generic components that are presentational. This is probably overkill as a refactor for this project right now but is a good principle here and going forward when thinking about keeping the codebase DRY.

For now, just extract these out and then make each one use the same presentational component.